### PR TITLE
Merge `workflow-deeplabcut`'s tutorial environment

### PR DESCRIPTION
**Pull request by MilagrosMarin**
## Description
- The [workflow-deeplabcut](https://github.com/datajoint/workflow-deeplabcut) repository provides a tutorial environment, notebooks, and pytests for running [element-deeplabcut](https://github.com/datajoint/element-deeplabcut).
- In the proposed changes, we integrate the contents of `workflow-deeplabcut` into `element-deeplabcut`, and will subsequently archive `workflow-deeplabcut`.
- We containerize the repository and update the setup

## Changes
- [x] Use `git merge` to merge the `workflow-deeplabcut` git history into `element-deeplabcut`
- [x] Subsequently made the following updates:
  - [x] Include the `.devcontainer` and `docker-compose-db.yaml` files
     - [x] Containerization of the repository tested both locally and in Codespaces 
  - [x] To mirror [Element-Calcium-Imaging](https://github.com/datajoint/element-calcium-imaging) repository:
     - [x] Update `.github`, `license`, `gitignore` ,`contributing` and `workflows` 
     - [x] add `code of conduct` and `codespell` 
     - [x] Update image names
  - [x] Update `README`
  - [x] Update `docs/src` 
  - [x] Update `setup` to:
     - [x] delete `requirements`
     - [x] update `author_email`
     - [x] Update `install_requires`
     - [x] Add `extra_requirements` to install the default DLC, for macOS users, to install DLC-GUI, and to install Elements.
        - [x] `default-dlc` and `elements` installation tested in Dev Container locally
        - [x] `dlc_apple_mchips` tested in a conda environment in macOS Ventura 13.6
     - [x] Add `setup.cfg`
  - [x] Update `__init__.py` in `element-deeplabcut` directory to add `ENV` variables and prefixes in DataJoint database

 ## The following tasks will be completed in the following PRs: 
  - [ ] Update tutorial notebook
  - [ ] Update GitHub Actions and tests
  - [ ] Add a pre-existing image of the latest `element-deeplabcut`
  - [ ] Update version and changelog

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: DataJoint Contribution Guideline
-    url: https://docs.datajoint.org/python/community/02-Contribute.html
+    url: https://datajoint.com/docs/community/contribute/
```
Comment (kushalbakshi) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
    url: https://datajoint.com/docs/about/contribute/
```

File: setup.py [hunk 2]
```diff
@@ -1,30 +1,50 @@
...
-    requirements = f.read().splitlines()
-
 with open(path.join(here, pkg_name, "version.py")) as f:
     exec(f.read())
 
 setup(
     name=pkg_name.replace("_", "-"),
     version=__version__,
-    description="DataJoint Element for Continuous Behavior Tracking via DeepLabCut",
+    description="DeepLabCut DataJoint Element",
```
Comment (kushalbakshi) [inline setup.py:16]:
```suggestion
    description="DataJoint Element for Continuous Behavior Tracking via DeepLabCut",
```

File: setup.py [hunk 3]
```diff
@@ -1,30 +1,50 @@
...
     long_description=long_description,
     long_description_content_type="text/markdown",
     author="DataJoint",
-    author_email="info@vathes.com",
+    author_email="info@datajoint.com",
     license="MIT",
     url=f'https://github.com/datajoint/{pkg_name.replace("_", "-")}',
-    keywords="neuroscience behavior deeplabcut datajoint",
+    keywords="neuroscience behavior pose-estimation science datajoint",
```
Comment (kushalbakshi) [inline setup.py:23]:
```suggestion
    keywords="neuroscience behavior deeplabcut pose-estimation science datajoint",
```
Let's leave deeplabcut in there

File: setup.py [hunk 4]
```diff
@@ -1,30 +1,50 @@
...
+            "tensorflow-metal",
+            "tables==3.7.0",
+            "deeplabcut",
+        ],
+        "dlc_gui": ["deeplabcut[gui]"],
+        "elements": [
+            "element-lab>=0.2.0",
+            "element-animal>=0.1.5",
+            "element-session>=0.1.2",
+            "element-interface>=0.5.0",
```
Comment (kushalbakshi) [inline setup.py:47]:
```suggestion
            "element-lab>=0.3.0",
            "element-animal>=0.1.8",
            "element-session>=0.1.5",
            "element-interface>=0.6.0",
```

**Review by kushalbakshi — CHANGES_REQUESTED** [review]
Looks good! Just a few small things.

**Review by MilagrosMarin — COMMENTED** [review]
I have accepted your suggestions, @kushalbakshi . Thank you!

**Review by kushalbakshi — APPROVED** [review]

**Comment by kushalbakshi** [comment]
Let's wait on the `Black` formatting until we are ready to merge this into the `main` branch. I was able to build a Codespace without any issues so I'll proceed with merging. Thanks for your work on this @MilagrosMarin

**Pull request merged**